controllers/consoleplugin/config/static-frontend-config.yaml

```diff
@@ -703,6 +709,11 @@ filters:
     component: autocomplete
     placeholder: 'E.g: Ingress, Egress, Inner'
     hint: Specify the direction of the Flow observed at the Node observation point.
+  - id: flow_type
+    name: Flow type
+    component: autocomplete
```

Comment:
To use `autocomplete` component you will need to implement `getOptions` on plugin side:
https://github.com/netobserv/network-observability-console-plugin/blob/be4ea4532962056f32080cd29bc99ce1fe77ca89/web/src/utils/filter-definitions.ts#L253-L297

We could even improve that to something more generic for `Namespaces`, `Names`, `Kinds` and this new `FlowType` since all of these rely on [autoCompleteCache](https://github.com/netobserv/network-observability-console-plugin/blob/eadc9beff97a2deef33d0494fc080ed3e31e7612/web/src/utils/autocomplete-cache.ts).

That would require a new route, sending field name as argument and refactor [resources.go](https://github.com/netobserv/network-observability-console-plugin/blob/0c2cebc3a144b9e3c24ad266f9af826e5b5aedad/pkg/handler/resources.go) accordingly. WDYT ?

Reply:
I changed to text. I think this is enough since the placeholder already hint the user that this is only infra or app.

And I agree that as a follow up to the generic column, we could have a follow up task to have generic auto complete for generic column.